### PR TITLE
[docs] Add cryptography examples

### PR DESCRIPTION
**Pull request by hhunter-ms**
Add `encrypt` and `decrypt` examples to the go client doc
Remove empty headers that still need examples someday :)

### CODE REVIEW

File: daprdocs/content/en/go-sdk-docs/go-client/_index.md [hunk 1]
```diff
@@ -145,10 +145,6 @@ err := testClient.ExecuteStateTransaction(ctx, store, meta, ops)
 For a full guide on state management, visit [How-To: Save & get state]({{< ref howto-get-save-state.md >}}).
 
 
-### Query state
-
```
Comment (daixiang0) [inline daprdocs/content/en/go-sdk-docs/go-client/_index.md:149]: Please add TODO not remove it directly.

File: daprdocs/content/en/go-sdk-docs/go-client/_index.md [hunk 2]
```diff
@@ -187,7 +183,6 @@ out, err := client.InvokeBinding(ctx, in)
 
 For a full guide on output bindings, visit [How-To: Use bindings]({{< ref howto-bindings.md >}}).
 
-### Actors
```
Comment (daixiang0) [inline daprdocs/content/en/go-sdk-docs/go-client/_index.md:190]: Same as above.

File: daprdocs/content/en/go-sdk-docs/go-client/_index.md (hunk 1 above)
Comment (hhunter-ms) [reply daprdocs/content/en/go-sdk-docs/go-client/_index.md:149]: @daixiang0 Until we have content for that section, I'd rather remove completely from the docs - unless you have a go-sdk example for query state that I can use here?

File: daprdocs/content/en/go-sdk-docs/go-client/_index.md (hunk 2 above)
Comment (hhunter-ms) [reply daprdocs/content/en/go-sdk-docs/go-client/_index.md:190]: @daixiang0 Until we have content for that section, I'd rather remove completely from the docs - unless you have a go-sdk example for actors that I can use here?

**Comment by daixiang0** [comment]
@hhunter-ms please make CI happy then we can make it land.

File: daprdocs/content/en/go-sdk-docs/go-client/_index.md [hunk 3]
```diff
@@ -293,6 +288,36 @@ go func() {
 
 For a full guide on configuration, visit [How-To: Manage configuration from a store]({{< ref howto-manage-configuration.md >}}).
 
+### Cryptography
+
+With the Dapr client Go SDK, you can use the high-level `Encrypt` and `Decrypt` cryptography APIs to encrypt and decrypt files while working on a stream of data.
```
Comment (msfussell) [inline daprdocs/content/en/go-sdk-docs/go-client/_index.md:293]:
```suggestion
With the Dapr client Go SDK, you can use the `Encrypt` and `Decrypt` cryptography APIs to encrypt and decrypt files while working on a stream of data.
```

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**